docs/csharp/getting-started/with-visual-studio.md

```diff
@@ -12,78 +12,74 @@ ms.devlang: csharp
+   This code displays "What is your name?" in the console window and waits until the user enters a string followed by the Enter key. It stores this string to a variable named `name`. It also retrieves the value of the @System.DateTime.Now property, which contains the current local time, and assigns it to a variable named `date`. Finally, it uses a [composite format string](../../standard/base-types/composite-format.md) to display these values to the console.
 
-2. Compile the program by choosing **Build** > **Build Solution**. This compiles your program to IL, an intermediate language that is then converted to binary code by a just-in-time (JIT) compiler.
+1. Compile the program by choosing **Build** > **Build Solution**.
 
-3. Run the program in debug mode in Visual Studio by selecting the green arrow on the toolbar, pressing F5, or choosing the **Debug** > **Start Debugging** menu item. After you respond to the prompts by entering a name and pressing the Enter key, the console window should look something like the following:
+1. Run the program in Debug mode in Visual Studio by selecting the green arrow on the toolbar, pressing F5, or choosing the **Debug > Start Debugging** menu item. Respond to the prompt by entering a name and pressing the Enter key.
 
-   ![The modified program running](./media/with-visual-studio/console.jpg)
+   ![Console window with modified program output](./media/with-visual-studio/helloworld2.png)
 
-4. Press any key to close the console window. This ends debug mode.
+1. Press any key to close the console window.
 
-You've now created and run your simple application. To develop a professional application, there are still some additional steps that you can take to make your application ready for release:
+You've created and run your application. To develop a professional application, take some additional steps to make your application ready for release:
 
 - For information on debugging your application, see [Debugging the Hello World Application](debugging-with-visual-studio.md)
 
 - For information on developing a publishing a distributable version of your application, see [Publishing the Hello World application](publishing-with-visual-studio.md).
 
-## Related topics ##
+## Related topics
 
 Instead of a console application, you can also build a class library with .NET Core and Visual Studio 2017. For a step-by-step introduction, see [Building a class library with C# and .NET Core in Visual Studio 2017](library-with-visual-studio.md).
 
-You can also develop a .NET Core console app on Mac, Linux, and Windows by using Visual Studio Code, a freely downloadable code editor. For a step-by-step tutorial, see [Getting Started with Visual Studio Code](with-visual-studio-code.md).
+You can also develop a .NET Core console app on Mac, Linux, and Windows by using Visual Studio Code, a downloadable code editor. For a step-by-step tutorial, see [Getting Started with Visual Studio Code](with-visual-studio-code.md).
```

Comment:
@rpetrusha The style manual flags "free" wording: Although this isn't advertising and marketing, I guessed (perhaps incorrectly) that the legal guidelines still apply. For example if the free item is a download and listed as being "free," we're supposed to say, "Connect time charges may apply." They have a six month rule, where they say a product or service can't be listed as "free" past six months. They want a link to terms and conditions. Idk how serious they would be about this, but I felt that just removing the word fixes the problem.